R/SCE2AnnData.R

```diff
@@ -208,7 +206,7 @@ SCE2AnnData <- function(sce, X_name = NULL, assays = TRUE, colData = TRUE,
         }
         cli::cli_progress_done()
     }
-    adata$uns <- reticulate::dict(uns_list)
+    adata$uns <- r_to_py(uns_list)
```

Comment:
I assume you still end up with a dictionary from this? I think I enforced somewhere that the list has to be named so I think this should be ok.